xbmc/platform/darwin/peripherals/Input_Gamecontroller.mm

```diff
@@ -155,14 +155,14 @@ - (void)registerChangeHandler:(GCController*)controller
     CLog::Log(LOGDEBUG, "INPUT - GAMECONTROLLER: microGamepad changehandler added");
     [self microValueChangeHandler:controller];
   }
-  if (@available(iOS 13.0, tvOS 13.0, *))
+  if (@available(iOS 13.0, tvOS 13.0, macOS 10.15, *))
```

Comment:
in this case correct way would be to use `NSProcessInfo` to determine OS version at runtime, as `@available` is used for new API checks

Reply:
As discussed, seems cleaner to use @available for now.